indico/modules/rb/services/rooms.py

```diff
@@ -99,11 +102,16 @@ def _checkParams(self):
         blocking_id = self._params.get('blocking_id')
         self._room = Room.get(self._params['room_id'])
         self._blocking = Blocking.get(blocking_id) if blocking_id else None
+        self._start_dt = datetime.strptime(self._params.get('start_dt'), '%H:%M %Y-%m-%d')
+        self._end_dt = datetime.strptime(self._params.get('end_dt'), '%H:%M %Y-%m-%d')
+        self._nonbookable = bool(NonBookablePeriod.find_first(NonBookablePeriod.room_id == self._room.id,
```

Comment:
This whole thing breaks the behaviour of the method when called from [here](https://github.com/indico/indico/blob/v1.9.1/indico/htdocs/js/indico/RoomBooking/util.js#L34), since there are no dates.